common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -105,9 +108,10 @@ trait Logging {
       val context = new java.util.HashMap[String, String]()
 
       args.foreach { mdc =>
-        sb.append(mdc.value.toString)
+        val value = if (mdc.value != null) mdc.value.toString else null
```

Comment:
@panbingkun nice catch. What will the null become in the log output?

Reply:
I added a case like this in UT.

Reply:
I mean in the log file or console. I wonder if we should change null value as empty string `""`

Reply:
ok, I tried it. Looks fine to me.